src/Uno.UI-Skia-only.slnf

```diff
@@ -54,6 +54,7 @@
       "Uno.UI.Toolkit\\Uno.UI.Toolkit.Skia.csproj",
       "Uno.UI.XamlHost.Skia.Wpf\\Uno.UI.XamlHost.Skia.Wpf.csproj",
       "Uno.UI.XamlHost\\Uno.UI.XamlHost.Skia.csproj",
+      "Uno.UI\\Uno.UI.Reference.csproj",
```

Comment:
Please also add `Uno.Reference` and `Uno.Foundation.Reference` otherwise browsing code in `Uno.UI` makes VS always complain about missing types when the files are viewed without explicitly selecting Skia target